modules/create_timepoint/php/ajaxtimepoint.class.inc

```diff
@@ -0,0 +1,302 @@
+<?php
+/**
+ * POST request methods.
+ *
+ * Used to retrieve info & create timepoint by the create_timepoint form.
+ * Ensures the user can create timepoint before processing
+ * the POST body (array) data.
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Create_Timepoint
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\create_timepoint;
+
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+
+/**
+ * The NDB_Page for the POST requests.
+ *
+ * @category Loris
+ * @package  Create_Timepoint
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class AjaxTimepoint extends \NDB_Page
+{
+    public $skipTemplate = true;
+
+    /**
+     * This function will return a json file with DCCID and PSCID
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request
+     *
+     * @return ResponseInterface The outgoing PSR7 response
+     * @throws \DatabaseException
+     * @throws \DeprecatedException
+     * @throws \LorisException
+     */
+    public function handle(ServerRequestInterface $request) : ResponseInterface
+    {
+        /**
+         * Ensure it's a POST request.
+         */
+        if ($request->getMethod() != 'POST') {
+            return (new \LORIS\Http\Response())
+                ->withHeader('Content-Type', 'application/json')
+                ->withStatus(405)
+                ->withHeader('Allow', 'POST')
+                ->withBody(
+                    new \LORIS\Http\StringStream(
+                        'Only POST requests are allowed'
+                    )
+                );
+        }
+
+        /**
+         * Parse POST request body.
+         */
+        $postResponse = $request->getParsedBody();
```

Comment:
`postResponse` name do not fit it's content. `requestBody` or `requestParams` maybe?